client/src/components/PageHistoryViewer/PageHistoryViewer.js

```diff
@@ -0,0 +1,12 @@
+import { compose } from 'redux';
+import readOnePageQuery from 'state/history/readOnePageQuery';
+
+/**
+ * Transformer function which binds the readOnePageQuery GraphQL query to the HistoryViewer
+ * component.
+ */
+const withPagesHistoryViewer = (HistoryViewer) => compose(
+  readOnePageQuery
+)(HistoryViewer);
```

Comment:
this reads really weird to me, isn't this the equivalent of just `const withPagesHistoryViewer = readOnePageQuery;`?

Don't get me wrong, great work :D I'm just curious why all the wrapping

Reply:
Yup, same thing as with the [silverstripe/elemental PR](https://github.com/dnadesign/silverstripe-elemental/pull/207), the whole HOC can be eliminated.

```js
      updater.component(
        'HistoryViewer.pages-controller-cms-content',
        readOnePageQuery,
        'PageHistoryViewer'
      );
```

Thanks for catching this. Didn't realise there were two of them.